llvm/test/MC/X86/register-assignment-error.s

```diff
@@ -0,0 +1,8 @@
+// RUN: not llvm-mc -triple x86_64-unknown-unknown %s -o /dev/null 2>&1 | FileCheck %s --check-prefix=CHECK-ERROR
```

Comment:
Minor suggestion

You don't need `-unknown-unknown`, `x86_64` is enough. Ony one check prefix is used, `--check-prefix=CHECK-ERROR` is redundant.